earth_enterprise/BUILD_RPMS.md

```diff
@@ -0,0 +1,70 @@
+# Installing via Package Managers
```

Comment:
We need a section or paragraph describing how to un-install via Package Managers..